### DISCUSSION TITLE
Unexpected behavior re: list style in a Gatsby site

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [tailwindlabs/tailwindcss#9062] [Help] [answered]
I came across [List Style Types](https://tailwindcss.com/docs/list-style-type) in the docs and wanted to give it a try on my GatsbyJS project. 

I attempted to implement this way: 

```
         <ul className="list-disc">
                  <li>Tuesdays - September 13, 2022 - June 6, 2023 - 4:00 - 6:00 pm</li>
                  <li>Wednesdays - September 14,2022  - June 7, 2023 - 10:00 am - 12:00 pm</li>
                  <li>Thursdays - September 15, 2022 - June 8, 2023 - 4:00 - 6:00 pm</li>
          </ul>
```

but the style did not change at all.

So, after a bit or reading I came across [this](https://github.com/tailwindlabs/tailwindcss/issues/5385) bug report and so I gave the `typography` plugin a try. 

```
         <ul className="prose list-disc">
                  <li>Tuesdays - September 13, 2022 - June 6, 2023 - 4:00 - 6:00 pm</li>
                  <li>Wednesdays - September 14,2022  - June 7, 2023 - 10:00 am - 12:00 pm</li>
                  <li>Thursdays - September 15, 2022 - June 8, 2023 - 4:00 - 6:00 pm</li>
          </ul>
```

Now my list is indented and the font is gray but there is no bullet (*) icon as the docs show... 

Two questions: what am I doing wrong in regards to the list style types not rendering correctly? Also, I thought the `typography` plugin was for markdown content (or CMSs) which my pages are not. So, do I really need the `typography` plugin?

**Comment by cannap** [comment]
hello, show code the color you have to set its by default gray

**Comment by wongjn** [comment, marked as answer]
`list-disc` works as intended and adds disc style bullets, see this [Tailwind Play](https://play.tailwindcss.com/jvB9ofdOye). Note that I had to wrap it with some padding so you could see the bullets, due to the fact that Tailwind's preflight resets `list-style-position` to its `initial` value, `outside`. Could it be a problem with your Tailwind not scanning the content that your example resides in?

As a side note even though you do not need the `@tailwindcss/typography` plugin, I thought I'd mention that the `prose` class should be a wrapper, like so:
```html
<div class="prose">
  <ul>
    <li>Tuesdays - September 13, 2022 - June 6, 2023 - 4:00 - 6:00 pm</li>
    <li>Wednesdays - September 14,2022  - June 7, 2023 - 10:00 am - 12:00 pm</li>
    <li>Thursdays - September 15, 2022 - June 8, 2023 - 4:00 - 6:00 pm</li>
  </ul>
</div>
```
[See this Tailwind Play](https://play.tailwindcss.com/yASwMP8Dns) to see how it looks.

**Reply by Isaac-Tait** [reply]
@wongjn I added the padding and the bullets appeared. They were there the whole time I just could not see them. Thank you 😄
